arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -677,6 +677,37 @@ def rule_gfm_1_3_8(
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_10(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.10] Remove the duplicate link:roleType element.
+    """
+
+    for modelDocument in val.modelXbrl.urlDocs.values():
+        if pluginData.isStandardTaxonomyUrl(modelDocument.uri, val.modelXbrl):
+            continue
+        rootElt = modelDocument.xmlRootElement
+        for element in rootElt.iterdescendants(tag="{http://www.xbrl.org/2003/linkbase}roleType"):
+            if isinstance(element,ModelObject):
+                roleURI = element.get("roleURI")
+                if roleURI is not None:
+                    modelRoleTypes = val.modelXbrl.roleTypes.get(roleURI)
+                    if modelRoleTypes and len(modelRoleTypes) > 1:
+                        yield Validation.warning(
+                            codes='EDINET.EC5700W.GFM.1.3.10',
+                            msg=_("Remove the duplicate link:roleType element."),
```

Comment:
We should include the URI in the message so that users from the GUI know which role is duplicated.